core/coreunix/test/metadata_test.go

```diff
@@ -189,6 +190,14 @@ func addDirectoryToBtfs(node *core.IpfsNode, file files.Node, metadata string) (
 	adder.Out = output
 	if metadata != "" {
 		adder.TokenMetadata = metadata
+		//
+		if _, ok := file.(files.Directory); ok {
+			adder.MetaForDirectory = true
+		}
+	}
+	if rs {
+		dsize, psize, csize := 10, 20, 262144
```

Comment:
small sync with other tests: `dsize, psize, csize := TestRsDataSize, TestRsParitySize, 262144`

Reply:
ok. will update. thank you.